mypyc/test-data/lowering-int.test

```diff
@@ -365,7 +365,6 @@ L2:
     r6 = r0 * 8
     r7 = r5 + r6
     r8 = load_mem r7 :: builtins.object*
-    inc_ref r8
     r9 = unbox(int, r8)
     dec_ref r8
```

Comment:
`inc_ref r8` is removed but `dec_ref r8` stays, is that intended? there's no borrow here so it seems like the `inc_ref` should stay.

Reply:
`LoadMem` now implicitly increfs the result if it has a reference counted type, similar to `GetAttr` and others, unless the borrow flag is true. See this code in `emitfunc.py`:
```py
    def visit_load_mem(self, op: LoadMem) -> None:
        dest = self.reg(op)
        src = self.reg(op.src)
        # TODO: we shouldn't dereference to type that are pointer type so far
        type = self.ctype(op.type)
        self.emit_line(f"{dest} = *({type} *){src};")
        if not op.is_borrowed:
            self.emit_inc_ref(dest, op.type)   # <<--- HERE
```

Reply:
ah right so that doesn't show up in the IR. thanks!